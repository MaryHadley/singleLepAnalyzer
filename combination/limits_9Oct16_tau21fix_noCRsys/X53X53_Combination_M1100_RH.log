000 (1297359.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297359.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.241:42920>
...
006 (1297359.000.000) 10/10 19:27:27 Image size of job updated: 155452
	50  -  MemoryUsage of job (MB)
	50728  -  ResidentSetSize of job (KB)
...
006 (1297359.000.000) 10/10 19:32:28 Image size of job updated: 283644
	157  -  MemoryUsage of job (MB)
	160592  -  ResidentSetSize of job (KB)
...
006 (1297359.000.000) 10/10 20:17:32 Image size of job updated: 284456
	160  -  MemoryUsage of job (MB)
	163288  -  ResidentSetSize of job (KB)
...
005 (1297359.000.000) 10/10 20:20:01 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:52:03, Sys 0 00:00:03  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:52:03, Sys 0 00:00:03  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1631  -  Run Bytes Sent By Job
	323  -  Run Bytes Received By Job
	1631  -  Total Bytes Sent By Job
	323  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1101        1   3824522
	   Memory (MB)          :      160     3072      3072
...
