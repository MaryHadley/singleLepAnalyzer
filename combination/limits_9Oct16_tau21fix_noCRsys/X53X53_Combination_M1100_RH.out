/var/lib/condor/execute/dir_5648
Total number of likelihood evaluations: 55000101
Total number of likelihood evaluations: 5500101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1100_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1100_RH/quant-toys:0-sig-4f4eab1f4c.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1100_RH/quant-data-sig-45427dc6e7.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1100_RH
