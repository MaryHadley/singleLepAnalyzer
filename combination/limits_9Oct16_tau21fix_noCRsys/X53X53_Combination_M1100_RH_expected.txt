# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.071198   0.036187    0.15159  0.049073    0.10559
